src/libAtomVM/term.c

```diff
@@ -379,3 +379,64 @@ int term_compare(term t, term other, Context *ctx)
 
     return result;
 }
+
+term term_alloc_refc_binary(Context *ctx, size_t size, unsigned is_const)
```

Comment:
let's change is_const type to bool. I don't think that `unsigned` alone is standard C.